modules/installation-configuration-parameters.adoc

```diff
@@ -80,6 +81,10 @@ endif::[]
 ifeval::["{context}" == "installing-restricted-networks-aws-installer-provisioned"]
 :aws:
 endif::[]
+ifeval::["{context}" == "installing-aws-outposts-remote-workers"]
+:aws:
+:aws-outposts:
```

Comment:
You do not need `aws-outposts` here. There is no content specific to Outposts in this file. `aws` will ensure that AWS specific parameters appear in your published output.

Reply:
:aws-outposts: added to line 65

Reply:
@ogradyp Please see my previous comment. Installation configuration parameters does not have any content that is specific to AWS Outposts. The only attribute that is required for `ifeval::["{context}" == "installing-aws-outposts-remote-workers"]` is `aws`. The latter attribute will ensure that the AWS content appears in your assembly.

Reply:
:aws-outposts: and :!aws-outposts: both removed